src/main/java/co/com/bancolombia/Constants.java

```diff
@@ -9,16 +9,16 @@
   public static final String APP_SERVICE = "app-service";
   public static final String PATH_GRAPHQL = "/graphqlpath";
   public static final String SECRETS_VERSION = "3.0.0";
-  public static final String SPRING_BOOT_VERSION = "2.6.2";
-  public static final String UNDERTOW_VERSION = "2.6.2";
-  public static final String JETTY_VERSION = "2.6.2";
+  public static final String SPRING_BOOT_VERSION = "2.6.6";
+  public static final String UNDERTOW_VERSION = "2.6.6";
+  public static final String JETTY_VERSION = "2.6.6";
   public static final String SONAR_VERSION = "3.0";
   public static final String LOMBOK_VERSION = "1.18.22";
   public static final String JACOCO_VERSION = "0.8.5";
   public static final String COBERTURA_VERSION = "3.0.0";
   public static final String RCOMMONS_ASYNC_COMMONS_STARTER_VERSION = "1.0.4";
   public static final String RCOMMONS_OBJECT_MAPPER_VERSION = "0.1.0";
-  public static final String PLUGIN_VERSION = "2.2.3";
+  public static final String PLUGIN_VERSION = "2.2.4";
```

Comment:
this line is change by a pipeline of github actions when the release is launched, the number is take from the tag